helpdesk_mgmt/models/helpdesk_ticket_category.py

```diff
@@ -5,7 +5,9 @@ class HelpdeskCategory(models.Model):
 
     _name = "helpdesk.ticket.category"
     _description = "Helpdesk Ticket Category"
+    _order = "sequence, id"
 
+    sequence = fields.Integer(default=1)
```

Comment:
Saner default (for all):

```suggestion
    sequence = fields.Integer(default=10)
```